### PR TITLE
Remove unused image assets

### PR DESCRIPTION
**Pull request by janfaracik**
### Proposed changelog entries

* Remove superfluous user interface elements, including images, `mask-icon`, and unnecessary resize handles.

### What's changed?
* Removed a fair few unused assets from Jenkins/combined assets where possible to cut down on space
* Removed `mask-icon` - https://www.leereamsnyder.com/blog/favicons-in-2021 - Jenkins now has a lovely colourful icon on Safari ❤️ 
* Textareas no longer have superfluous resize handles on browsers that support `resize` (anything that's not IE11)

### Screenshots

Old
image

New
image

---

Old 
image

New 
image

### Maintainer checklist

Before the changes are marked as `ready-for-merge`: 

- [ ] There are at least 2 approvals for the pull request and no outstanding requests for change
- [ ] Conversations in the pull request are over OR it is explicit that a reviewer does not block the change
- [ ] Changelog entries in the PR title and/or `Proposed changelog entries` are correct
- [ ] Proper changelog labels are set so that the changelog can be generated automatically
- [ ] If the change needs additional upgrade steps from users, `upgrade-guide-needed` label is set and there is a `Proposed upgrade guidelines` section in the PR title. ([example](https://github.com/jenkinsci/jenkins/pull/4387))
- [ ] If it would make sense to backport the change to LTS, a Jira issue must exist, be a _Bug_ or _Improvement_, and be labeled as `lts-candidate` to be considered (see [query](https://issues.jenkins-ci.org/issues/?filter=12146)).

### CODE REVIEW

File: war/src/main/less/base/style.less
Comment (timja) [inline war/src/main/less/base/style.less:1262]: It seems that by removing this you have to scroll all the way to the right to resize now, which is not so obvious, possibly because the text area is so large.

It also seems that you can no longer horizontally resize which is probably fine I don't think we need that.

any thoughts?

**Comment by jglick** [comment]
Have you searched plugin sources to make sure there are no users of the small icons proposed for deletion?

**Comment by janfaracik** [comment]
> Have you searched plugin sources to make sure there are no users of the small icons proposed for deletion?

Had a scan through and there's a few that are in use by extremely old plugins (eg ten years old) that haven't had any updates. The only plugin that I found that has recent updates is https://github.com/jenkinsci/release-plugin - it's currently using the atom.gif icon, I'd be happy to make an MR for it to use the new RSS icon currently used by Jenkins

File: war/src/main/less/base/style.less
Comment (timja) [reply war/src/main/less/base/style.less:1262]: I think this will be fixed by making the form less wide

**Review by timja — COMMENTED** [review]
Looks good just needs release plugin sorted from what I can see

**Comment by janfaracik** [comment]
> Looks good just needs release plugin sorted from what I can see

Juts opened a PR for that - https://github.com/jenkinsci/release-plugin/pull/36 :)

**Review by timja — APPROVED** [review]

**Review by uhafner — APPROVED** [review]

**Review by fqueiruga — APPROVED** [review]
Nice. off topic, I think I should create a PR to remove the experimental header we introduced... 2 years ago 😅

**Review by fqueiruga — APPROVED** [review]
Nice. off topic, I think I should create a PR to remove the experimental header we introduced... 2 years ago 😅

**Comment by timja** [comment]
> Nice. off topic, I think I should create a PR to remove the experimental header we introduced... 2 years ago 😅

Yeah probably, not worth keeping that tech debt around since it was never enabled by default

----

This PR is now ready for merge, after ~24 hours, we will merge it if there's no negative feedback.

Thanks!

**Comment by daniel-beck** [comment]
Caused [JENKINS-69319](https://issues.jenkins.io/browse/JENKINS-69319).

**Pull request merged**
